driver/src/it/scala/org/mongodb/scala/DocumentationExampleSpec.scala

```diff
@@ -0,0 +1,516 @@
+      Document("""{ item: "planner", qty: 75, size: { h: 22.85, w: 30, uom: "cm" }, status: "D" }"""),
+      Document("""{ item: "postcard", qty: 45, size: { h: 10, w: 15.25, uom: "cm" }, status: "A" }"""),
+      Document("""{ item: "sketchbook", qty: 80, size: { h: 14, w: 21, uom: "cm" }, status: "A" }"""),
+      Document("""{ item: "sketch pad", qty: 95, size: { h: 22.85, w: 30.5, uom: "cm" }, status: "A" }""")
+    )).execute()
+    //End Example 51
+
+    collection.count().execute() should equal(10)
+
+    //Start Example 52
+    import org.mongodb.scala.model.Filters
+    import org.mongodb.scala.model.Updates
+    collection.updateOne(Filters.eq("item", "paper"),
+      Updates.combine(Updates.set("size.uom", "cm"), Updates.set("status", "P"), Updates.currentDate("lastModified"))
+    ).execute()
+    //End Example 52
+
+    collection.find(Filters.eq("item", "paper")).execute().foreach(doc => {
+      doc.get[BsonDocument]("size").get.get("uom") should equal(BsonString("cm"))
+      doc.get[BsonString]("status").get should equal(BsonString("P"))
+      doc.containsKey("lastModified") shouldBe true
+    })
+
+    //Start Example 53
+    collection.updateMany(Filters.lt("qty", 50),
+      Updates.combine(Updates.set("size.uom", "in"), Updates.set("status", "P"), Updates.currentDate("lastModified"))
+    ).execute()
+    //End Example 53
+
+    collection.find(Filters.lt("qty", 50)).execute().foreach(doc => {
+      doc.get[BsonDocument]("size").get.get("uom") should equal(BsonString("in"))
+      doc.get[BsonString]("status").get should equal(BsonString("P"))
+      doc.containsKey("lastModified") shouldBe true
+    })
+
+    //Start Example 54
+    collection.replaceOne(Filters.eq("item", "paper"),
+      Document("""{ item: "paper", instock: [ { warehouse: "A", qty: 60 }, { warehouse: "B", qty: 40 } ] }""")
+    ).execute()
+    //End Example 54
+
+    import org.mongodb.scala.model.Projections
+    collection.find(Filters.eq("item", "paper")).projection(Projections.excludeId()).execute().foreach(doc =>
+      doc should equal(Document("""{ item: "paper", instock: [ { warehouse: "A", qty: 60 }, { warehouse: "B", qty: 40 } ] }"""))
+    )
+  }
+  
+  it should "be able to delete" in withCollection { collection =>
+
+    //Start Example 55
+    collection.insertMany(Seq(
+      Document("""{ item: "journal", qty: 25, size: { h: 14, w: 21, uom: "cm" }, status: "A" }"""),
+      Document("""{ item: "notebook", qty: 50, size: { h: 8.5, w: 11, uom: "in" }, status: "A" }"""),
+      Document("""{ item: "paper", qty: 100, size: { h: 8.5, w: 11, uom: "in" }, status: "D" }"""),
+      Document("""{ item: "planner", qty: 75, size: { h: 22.85, w: 30, uom: "cm" }, status: "D" }"""),
+      Document("""{ item: "postcard", qty: 45, size: { h: 10, w: 15.25, uom: "cm" }, status: "A" }""")
+    )).execute()
+    //End Example 55
+
+    collection.count().execute() should equal(5)
+
+    //Start Example 56
```

Comment:
The examples can be out of order in this file so you don't have to re-insert all the sample documents. The docs team doesn't need them to be in order here.